packages/scene-composer/src/components/panels/AddComponentMenu.tsx

```diff
@@ -134,6 +157,15 @@ export const AddComponentMenu: React.FC<AddComponentMenuProps> = ({ onSelect })
     addComponentInternal(selectedSceneNodeRef, component);
   }, [selectedSceneNodeRef, selectedSceneNode]);
 
+  const handleAddAnimations = useCallback(() => {
+    if (!selectedSceneNodeRef) return;
+
+    let selector = animationComponent.selector || 0;
```

Comment:
Put a Asana task in the project to update the UX so we can remove selector from the Animation Component data model so we can fix this down the road.

Reply:
done

Reply:
https://app.asana.com/0/1204688230973005/1205234186255686/f